tests/s25Main/lua/testLuaSettings.cpp

```diff
@@ -149,6 +149,10 @@ BOOST_AUTO_TEST_CASE(Events)
     BOOST_TEST_REQUIRE(allowedAddons[0] == AddonId::LIMIT_CATAPULTS);
     BOOST_TEST_REQUIRE(allowedAddons[1] == AddonId::CHARBURNER);
     BOOST_TEST_REQUIRE(allowedAddons[2] == AddonId::TRADE);
+
+    executeLua("function isMapPreviewEnabled()\n  return false\nend");
+    BOOST_TEST_REQUIRE(!lua.IsMapPreviewEnabled());
```

Comment:
Also check the default:

```suggestion
    BOOST_TEST(lua.IsMapPreviewEnabled()); // Not defined in lua -> Default to enabled
    executeLua("function isMapPreviewEnabled()\n  return false\nend");
    BOOST_TEST(!lua.IsMapPreviewEnabled());
    executeLua("function isMapPreviewEnabled()\n  return true\nend");
    BOOST_TEST(lua.IsMapPreviewEnabled());
```

(Yes we erroneously used `BOOST_TEST_REQUIRE` for no good reason around this)